tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/AWS/LambdaCommon.cs

```diff
@@ -26,6 +28,18 @@ internal class LambdaCommon
         private const string PlaceholderOperationName = "placeholder-operation";
         private const string DefaultJson = "{}";
         private const double ServerlessMaxWaitingFlushTime = 3;
+        private static readonly DateTimeJsonConverter DateTimeConverter = new();
+        private static readonly MemoryStreamJsonConverter MemoryStreamConverter = new();
+        private static readonly ConverterContractResolver ContractResolver = new();
+        private static readonly JsonSerializerSettings SerializerSettings = new()
```

Comment:
Just nitpicking the code formatting, but we usually separate multi-line constructs with blank links.
```suggestion
        private static readonly ConverterContractResolver ContractResolver = new();

        private static readonly JsonSerializerSettings SerializerSettings = new()
```

Reply:
Thanks for letting me know, will update it!